packages/web-components/src/components/rux-checkbox/rux-checkbox.scss

```diff
@@ -65,11 +65,13 @@
                 align-self: start;
                 height: 1.125rem;
                 width: 1.125rem;
-
-                margin: 0 0.625rem 0 0;
+                margin: calc(var(--spacing-1) / 2) calc(var(--spacing-2) + 2px)
+                    calc(var(--spacing-1) / 2) calc(var(--spacing-1) / 2);
+                //margin: calc(var(--spacing-1) / 2);
                 background-color: var(--color-background-base-default);
-                border: 1px solid var(--color-border-interactive-muted);
-                border-radius: var(--radius-checkbox);
+                border: var(--border-width-xs) solid
+                    var(--color-border-interactive-muted);
+                border-radius: 2px;
```

Comment:
Should this be `var(--radius-base)`? I know checkbox is a bit weird so it may not be, just wanted to double check.

Reply:
It was but that variable appears to not exist so I swapped it back to 2px.

Reply:
Alrighty, well the `radius-base` is 3px anyway so I'm fine with this. good work!